### ISSUE TITLE
LegacyComponentSerializerImpl exception

### ISSUE THREAD

**Issue by Syntebyte** [PaperMC/Paper#5240]
### What behaviour is expected:

ImageOnMap plugin has feature to rename your maps.
You can click on book and write new name on sign.

![Screenshot_23443](https://user-images.githubusercontent.com/26767782/108913223-bca45880-763a-11eb-96db-433778ecbeb3.png)
![Screenshot_4](https://user-images.githubusercontent.com/26767782/108914078-eb6efe80-763b-11eb-93a9-71127dd1a9ff.png)
![Screenshot_5](https://user-images.githubusercontent.com/26767782/108914086-edd15880-763b-11eb-9e07-9f93bc4dfdfe.png)

**Works fine on Paper 468!**

### What behaviour is observed:
I've updated Paper jar to git-Paper-484, then try latest git-Paper-491
Nothing happens after book click and ImageOnMap keeps throw NullPointerException:
```
[00:25:04 WARN]: [ImageOnMap] Task #3471 for ImageOnMap v4.0 generated an exception
--
  | java.lang.NullPointerException: null
  | at net.kyori.adventure.text.serializer.legacy.LegacyComponentSerializerImpl.deserialize(LegacyComponentSerializerImpl.java:205) ~[patched_1.16.5.jar:git-Paper-484]
  | at org.bukkit.craftbukkit.v1_16_R3.block.CraftSign.setLine(CraftSign.java:74) ~[patched_1.16.5.jar:git-Paper-484]
  | at fr.moribus.imageonmap.components.gui.PromptGui.setSignContents(PromptGui.java:261) ~[?:?]
  | at fr.moribus.imageonmap.components.gui.PromptGui.open(PromptGui.java:176) ~[?:?]
  | at fr.moribus.imageonmap.components.gui.Gui$1.run(Gui.java:112) ~[?:?]
  | at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftTask.run(CraftTask.java:99) ~[patched_1.16.5.jar:git-Paper-484]
  | at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:468) ~[patched_1.16.5.jar:git-Paper-484]
  | at net.minecraft.server.v1_16_R3.MinecraftServer.b(MinecraftServer.java:1294) ~[patched_1.16.5.jar:git-Paper-484]
  | at net.minecraft.server.v1_16_R3.DedicatedServer.b(DedicatedServer.java:377) ~[patched_1.16.5.jar:git-Paper-484]
  | at net.minecraft.server.v1_16_R3.MinecraftServer.a(MinecraftServer.java:1209) ~[patched_1.16.5.jar:git-Paper-484]
  | at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:997) ~[patched_1.16.5.jar:git-Paper-484]
  | at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:174) ~[patched_1.16.5.jar:git-Paper-484]
  | at java.lang.Thread.run(Unknown Source) [?:1.8.0_261]
```

### Steps/models to reproduce:
1. Run Paper-1.16.5-491 with ImageOnMap 4.0 https://dev.bukkit.org/projects/imageonmap/files/2980012
2. Make maps with **/tomap https://i.imgur.com/MZUV4Bl.jpeg**
3. Type **/maps** for open maps menu
4. Hover cursor on map and Right-Click
5. Left-Click on book "Rename this image"
6. Nothing happened. Exception in console throwed.
> You can try it with Paper 468. Works fine. ^

### Plugin list:
![Screenshot_3](https://user-images.githubusercontent.com/26767782/108911247-12c3cc80-7638-11eb-8db2-6e9936cecd61.png)

### Paper version:
Output of **/version**:
```
[00:33:35 INFO]: This server is running Paper version git-Paper-491 (MC: 1.16.5) (Implementing API version 1.16.5-R0.1-SNAPSHOT)
[00:33:35 INFO]: Previous version: git-Paper-484 (MC: 1.16.5)
[00:33:35 INFO]: You are running the latest version
```
### P. S.
I think the recent "Adventure" update might be causing this problem https://github.com/PaperMC/Paper/pull/4842

**Comment by jpenilla** [comment]
Please try build 492

**Comment by kashike** [comment]
Should be fixed with https://github.com/PaperMC/Paper/commit/aa282a1a048a82dc1b1025f862e8cd38789c9e3a.

**Comment by Syntebyte** [comment]
> Should be fixed with [aa282a1](https://github.com/PaperMC/Paper/commit/aa282a1a048a82dc1b1025f862e8cd38789c9e3a).

> Please try build 492

Successfuly fixed! Thanks. awesome
